protocol/lib/quantums.go

```diff
@@ -63,7 +63,7 @@ func BaseToQuoteQuantums(
 //	  quoteQuantums / priceValue /
 //	  10^(priceExponent + baseCurrencyAtomicResolution - quoteCurrencyAtomicResolution)
 //
-// The result is rounded down.
+// The result is rounded down. It is okay to do the divisions separately, as the result is the same.
```

Comment:
https://math.stackexchange.com/questions/2557458/is-a-b-c-equal-to-a-bc-for-integer-division#:~:text=Furtrher%20we%20already%20know%20that,b%20and%20a%20remainder%20q.